Rakefile

```diff
@@ -5,11 +5,8 @@ task default: ["notify:expired"]
 
 namespace :notify do
   pages_urls = [
-    "https://www.docs.verify.service.gov.uk/api/pages.json",
     "https://gds-way.cloudapps.digital/api/pages.json",
```

Comment:
I would update this to https://gds-way.digital.cabinet-office.gov.uk/api/pages.json at the same time

Reply:
Great shout, done.